schema/table.go

```diff
@@ -76,6 +76,9 @@ type Table struct {
 	// documented.
 	IsIncremental bool `json:"is_incremental"`
 
+	// IsPaid is a flag that indicates if the table is a paid resource or not.
+	IsPaid bool `json:"-"`
```

Comment:
This field will be used by the plugin before sending the sync message, so doesn't need to be passed as metadata.

Reply:
I think we do want it to be part of the JSON output, so should include ``json:"is_paid"`` here

For the `cloudquery tables` command to use it it will also need to be part of the Arrow metadata, but this is not too important right now

Reply:
Good point, modified in https://github.com/cloudquery/plugin-sdk/pull/1319/commits/ca9cedf2a4fe2adf189af1551b9a6c7cd744ea23 and tested locally using `cloudquery cli` - which needs changes in a follow-up PR